spec/haskell/src/SEL4/Object/Structures.lhs

```diff
@@ -516,3 +513,6 @@ Sets of TCB flags are bitwise OR'd and represented as a word.
 
 > isFlagSet :: TcbFlag -> TcbFlags -> Bool
 > isFlagSet flag flags = tcbFlagToWord flag .&. flags /= 0
+
+> tcbFlagMask :: Word
+> tcbFlagMask = if config_HAVE_FPU then tcbFlagToWord FpuDisabled else 0
```

Comment:
if we were writing code only, I'd suggest noFlag here instead of 0. Incidentally, the 0x0 above can be 0, but I'm fine to leave both.